wrappers/python/simtk/openmm/app/modeller.py

```diff
@@ -715,6 +715,9 @@ def addHydrogens(self, forcefield=None, pH=7.0, variants=None, platform=None):
         Definitions for standard amino acids and nucleotides are built in.  You can call loadHydrogenDefinitions() to load
         additional definitions for other residue types.
 
+        By default, except when variants is not set, existing hydrogen atoms in the structure are kept and their positions
+        are not optimized.
```

Comment:
This sentence is kind of confusing.  `variants` not being set *is* the default.  Also, I don't see where it treats them differently if `variants` is set?

Reply:
Too much passive voice alright... My understanding of the `variants` option is that, e.g., if you have a HIE and you specify HID, that epsilon proton is going to be removed, thus affecting an existing atom.

How about: `Only the positions of newly added hydrogen atoms will be optimized.` ?

Reply:
`variants` can affect which hydrogens get added or removed, but it doesn't affect which ones get optimized.  Also, users won't know what "optimized" refers to here.  That's an implementation detail.

How about adding a sentence just before the last paragraph: "In all cases, the positions of existing hydrogens are not modified."

Reply:
Sounds good!